docs/for_pandas.rst

```diff
@@ -0,0 +1,69 @@
+For Pandas
+==========
+
+Gokart has several features for Pandas.
+
+
+Pandas Type Config
+------------------
+
+Pandas often estimates and changes the type automatically. In gokart, specify a column name to check the type of Task input and output.
+
+
+.. code:: python
+
+    from typing import Any, Dict
+    import pandas as pd
+    import gokart
+
+
+    # Please define a class which inherits `gokart.PandasTypeConfig`.
+    class SamplePandasTypeConfig(gokart.PandasTypeConfig):
+
+        @classmethod
+        def type_dict(cls) -> Dict[str, Any]:
+            return {'int_column': int}
+
+
+    class SampleTask(gokart.TaskOnKart):
+
+        def run(self):
+            # [PandasTypeError] because expected type is `int`, but `str` is passed.
+            df = pd.DataFrame(dict(int_column=['a']))
+            self.dump(df)
+
+This is useful when developing batches.
```

Comment:
What is some example? 🤔

Reply:
I think gokart is also "useful for develping batches", this sentence should be a good example for type conversion issue.
How about `This is useful when dataframe has nullable columns because pandas auto-conversion often fails in such case`？

Reply:
Sounds good. Thanks.